### ISSUE TITLE
Landmark spatial interpretation?

### ISSUE THREAD

**Issue by waleedrazakhan92** [KupynOrest/head_detector#29]
Is there any map which tells which landmark number lands on what part of the face just like mediapipe?
I'm trying to find volume of parts of the face(cheeks, nose, lips etc.) using 3d landmarks and that require knowing what landmark identifies what part of the face.

**Comment by emlcpfx** [comment]
Yes, the landmarks correspond to the vertex numbers on the 3d model. Lots of different ways to visualize that, or use a 3D DCC to do it.

**Comment by divyansshh** [comment]
I would also like to know the exact landmarks corresponding to each point such as(eyes, nose, lips, hair, neck) for easier understanding and usage.

**Comment by waleedrazakhan92** [comment]
Could you also explain the 3d landmark values. I'm trying to find volume of cheeks, lips etc using the 3d landmark mean values. But I've noticed that some of the z-index values are positive and some negative. Shouldn't all the values be either positive or all negative. Or is there a reference point with which these values are calculated?

**Comment by emlcpfx** [comment]
If you plot that in 3D, where you can see the xyz axis, wouldn’t you just see that the head is half in front and half behind on the z axis?

**Comment by KupynOrest** [comment]
Hi, this is the expected behavior: _The coordinate origin in FLAME space corresponds to the geometric center of the 3D head model_
